bugwarrior/services/youtrack.py

```diff
@@ -133,24 +123,16 @@ def __init__(self, *args, **kw):
         self.base_url = f'{scheme}://{self.config.host}:{port}'
         if self.config.incloud_instance:
             self.base_url += '/youtrack'
-        self.rest_url = self.base_url + '/rest'
+        self.rest_url = self.base_url + '/api'
 
         self.session = requests.Session()
         self.session.headers['Accept'] = 'application/json'
         if not self.config.verify_ssl:
             requests.packages.urllib3.disable_warnings()
             self.session.verify = False
 
-        password = self.get_password('password', self.config.login)
-        if not self.config.anonymous:
-            self._login(self.config.login, password)
-
-    def _login(self, login, password):
-        params = {'login': login, 'password': password}
-        resp = self.session.post(self.rest_url + "/user/login", params)
-        if resp.status_code != 200:
-            raise RuntimeError("YouTrack responded with %s" % resp)
-        self.session.headers['Cookie'] = resp.headers['set-cookie']
+        token = self.config.token
```

Comment:
We should use `self.get_password` like was previously done, which enables users to use [oracles](https://bugwarrior-docs.readthedocs.io/en/latest/common_configuration.html#password-management) to inject their password as an alternative to storing it in the configuration file, which would look like:

```py
token = self.get_password('token', self.config.login)
```